spec.bs

```diff
@@ -4215,8 +4215,8 @@ metrics collected from a single execution of a worklet function:
 </dl>
 
 A <dfn>per participant metrics</dfn> is a [=struct=] with the following [=struct/items=],
-representing metrics aggregated over everything a particular participant does in an auction,
-including all bids, scoring, etc. it does:
+representing metrics aggregated over everything a particular participant (e.g. bidder or seller)

```

Comment:
is it enough/clearer to say "representing metrics aggregated over a particular participant (e.g. bidder or seller):"? Trying to understand what "everything" is telling.

Reply:
Probably. What's also funky is that sometimes reporting is separate, sometimes is not.